ios/ItaliaApp/AppDelegate.m

```diff
@@ -81,4 +81,34 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
   [RCTPushNotificationManager didReceiveLocalNotification:notification];
 }
 
+- (void)applicationWillResignActive:(UIApplication *)application {
+
+     // Fill screen with our own colour
+    UIView *colourView = [[UIView alloc]initWithFrame:self.window.frame];
+    colourView.backgroundColor = [UIColor whiteColor];
+    colourView.tag = 1234;
```

Comment:
is this number arbitrary? can we add some comment describing what's going on here? perhaps some reference too?

Reply:
Yes it is arbitrary and is used here https://github.com/teamdigitale/io-app/pull/943/files#diff-8d59bb2633e5b2fda2c2d6853cc45662R103 to get a reference to the view.

Reply:
yes pls describe this mechanism with a short comment

Reply:
Done.